integration/vm_factory/vm_templating_test.sh

```diff
@@ -54,9 +54,21 @@ check_vm_template_factory() {
 	[ $res -eq 1 ] || die "template factory is not set up, missing state file"
 }
 
+clean_vm_template() {
+	res=$(mount | grep ${template_tmpfs_path} | wc -l)
+	if [ $res -ne 0 ]; then
+		echo "cleaning vm template"
+		{ umount ${template_tmpfs_path} ; res=$?; } || true
```

Comment:
Need to `umount` multiple times in case there are more than one template tmpfs mounted?

Reply:
> Need to `umount` multiple times in case there are more than one template tmpfs mounted?

Submit a new version that can handle it.